rdt/hyper_transformer.py

```diff
@@ -254,6 +284,21 @@ def transform(self, data):
             data = transformer.transform(data, drop=False)
 
         transformed_columns = self._subset(self._output_columns, data.columns)
+
+        if self._transform_nulls:
+            for field in data.columns:
```

Comment:
can we pull all of these chunks into their own methods? One for `fit`, one for `transform` and one for `reverse_transform`